src/rpc/rawtransaction.cpp

```diff
@@ -201,7 +201,10 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
     }
     bool fLocked = instantsend.IsLockedInstantSendTransaction(txid);
     bool fLLMQLocked = llmq::quorumInstantSendManager->IsLocked(txid);
-    entry.push_back(Pair("instantlock", fLocked || fLLMQLocked));
+    entry.push_back(Pair("instantlock", fLocked || fLLMQLocked || chainLock));
+    if (LogAcceptCategory("instantsend")) {
```

Comment:
Hmm, I'm not sure I like this "misuse" of logging categories. In the end we might end up having people rely on the `instantsend` debug category being active to enable this field in RPCs...

Reply:
I had the same concern -- but the only other option (IMO) is to have an explicit "debug" flag in addition to a "debug" logging category, which seems like overkill.

Reply:
Why not always add that field?

Reply:
I thought it could be confusing to have this field in "normal" mode, cause normally, you shouldn't be using this data. I could also use `if (fDebug)` as an alternative but the idea was to limit it to IS debugging only. Now that I see that it causes more questions than I expected, I guess it's better to simplify this :)